### ISSUE TITLE
Show subtitles in web table of contents

### ISSUE THREAD

**Issue by allisonpbrown** [pressbooks/ideas#450]
### Feature Description

Currently in the web table of contents, only the main title of a chapter and the chapter author shows. The book settings allow for 1st level headings to be shown or hide, but there are no options for displaying the subtitles that users are able to enter in the chapter metadata. The PDF output table of contents shows the subtitle, but not the web.

PDF output: 
![Screenshot 2023-01-11 at 11 19 32 AM](https://user-images.githubusercontent.com/36453118/211859355-85be4cd9-a229-4511-8d00-62d47234b45c.png)
Web output:
<img width="905" alt="Screenshot 2023-01-11 at 11 20 23 AM" src="https://user-images.githubusercontent.com/36453118/211859521-c2f0dca7-1da2-462b-ae49-c97b84ff9971.png">

### Feature Use Case

The current book has different types of chapters, and we used the subtitle field to communicate to the reader which chapter is which type, rather than bog down the title of the chapter with it's appropriate description. There could be a toggle to turn on and off the subtitle, but I'm guessing if people use that field they want it to show, so I don't know if appearance options are necessary, just showing the data if it has been entered.

### Other Notes

The subtitle field is a little useless if you can't view the data. Also please don't make it a collapsed subheading that has to be manually toggled like the 1st level heading option.

**Comment by SteelWagstaff** [comment]
hi @allisonpbrown this is a really interesting idea. What would be your desired outcome?

Option 1: if a chapter has a subtitle, always display it by default in the webboook ToC, matching what occurs now in PDF exports
Option 2: create a theme option for the webbook to allow authors to display chapter subtitles in the webbook ToC. This option would be off by default and could be opted into on a book by book basis
Option 3: other [please provide details?]

**Comment by allisonpbrown** [comment]
I would say Option 1! If someone uses the subtitle field, I believe they would want it to show.

**Comment by SteelWagstaff** [comment]
Still relevant. @cmurtagh I'd be interested in opening a PR for this if the team was open to it?

**Comment by cmurtagh** [comment]
Let's get @MichelleWeremczuk's take on this.

**Comment by MichelleWeremczuk** [comment]
Agreed—this would be a helpful change. I would keep the same typography formatting as the PDF.

**Comment by abbeykelder** [comment]
Adding a quick note here to reconfirm that this is a wanted update! We have a book I am working on right now that has several chapters with the same title but different subtitles (e.g. Problem Questions: [Topic]) and it would be helpful to not have them all appear as "Problem Questions" in the TOC.

**Comment by cmurtagh** [comment]
@SteelWagstaff Are you still interested in submitting a PR for this? I can look at scheduling it next sprint.

**Comment by SteelWagstaff** [comment]
Yes, definitely

**Comment by arzola** [comment]
This issue has been automatically closed as part of an organization-wide backlog review.

**Reason:** This issue was created 3y 1m ago with no updates in the last 9m.

If this is still relevant, please feel free to reopen it with updated context. We appreciate your contributions and want to ensure our issue tracker stays actionable.

**Learn more:** https://pressbooks.community/t/announcement-automatic-closure-of-old-github-tickets/4542
